anyway/parsers/infographics_data_cache_updater.py

```diff
@@ -13,6 +13,50 @@
 db = SQLAlchemy(app)
 
 
+def is_cache_eligible(news_flash):
```

Comment:
This function is prone to change in the future, maybe a wrapper function for this could be helpful when implementation changes, or a OOP kind of solution.

Reply:
I agree. Not sure what object it should belong to. Seems to me it is not an attribute of the NewFlash it self, but of our inforgaphics data implementation, but maybe I am going to far here?

Reply:
No, I think you are on point.
Could you create such object?
@atalyaalon what say you?

Reply:
@ziv17 @Mano3 I agree, it's a part of the infographics data, since this filtering of newsflashes is relevant for the infographics data.